### PR TITLE
Improve configuration/screensaver.md

### PR DESCRIPTION
**Pull request by andrum99**
Some improvements and corrections to the screen saver page, within the Raspberry Pi OS section of the docs.

### CODE REVIEW

File: configuration/screensaver.md [hunk 1]
```diff
@@ -1,28 +1,34 @@
 # Setting the screen saver/screen blanking
 
-## On the Console
+## On the console
 
-If you are using the Raspberry Pi solely on the console (no desktop GUI), you need to set the console blanking. The current setting, in seconds, can be displayed using
+By default, when running without a graphical desktop, Raspberry Pi OS will blank the screen after 10 minutes without user input: either mouse movement or keyboard input.
```
Comment (lurch) [inline configuration/screensaver.md:5]: Perhaps this sentence should be reworded to specifically mention Raspberry Pi OS Lite, and remove the confusing "by default" part?

File: configuration/screensaver.md [hunk 2]
```diff
@@ -1,28 +1,34 @@
...
+Here, `consoleblank` is a kernel parameter. In order to be permanently set, it needs to be defined on the kernel command line. To edit the kernel command line:
 
 ```
 sudo nano /boot/cmdline.txt
 ```
 
-Add `consoleblank=0` to turn screen blanking off completely, or edit it to set the number of seconds of inactivity before the console will blank. Note the kernel command line must be a single line of text.
+The file `/boot/cmdline.txt` contains a single line of text. Add `consoleblank=n` to have the console blank after `n` seconds of inactivity, for example `consoleblank=300` for 300 seconds = 5 minutes. To disable screen blanking, set `consoleblank=0`. Make sure that you add your `consoleblank` option to the single line of text already in the `cmdline.txt` file.
+
+You can also use the `raspi-config` tool to disable screen blanking. Note that the screen blanking setting in `raspi-config` also controls screen blanking when the graphical desktop is running.
```
Comment (lurch) [inline configuration/screensaver.md:20]: Perhaps this sentence would be less ambiguous if "screen blanking" was hyphenated like "screen-blanking" ?

File: configuration/screensaver.md [hunk 3]
```diff
@@ -1,28 +1,34 @@
...
+
+## On the Raspberry Pi desktop
 
-## On the Raspberry Pi Desktop
+By default, Raspberry Pi OS will blank the graphical desktop after 10 minutes without user input. You can disable this by changing the 'Screen Blanking' option on the Raspberry Pi Configuration tool, which is available on the Preferences menu. Note that the 'Screen Blanking' option also controls screen blanking when the graphical desktop is not running.
```
Comment (lurch) [inline configuration/screensaver.md:24]: "option on" -> "option in" ?

File: configuration/screensaver.md [hunk 4]
```diff
@@ -1,28 +1,34 @@
...
 
-By default, the Raspberry Pi Desktop does not have any easy-to-use screensaver software installed, although the screensaver is enabled. Firstly, you should install the X Windows screensaver application.
+There is also a graphical screen saver available, which can be installed as follows:
```
Comment (lurch) [inline configuration/screensaver.md:26]: Looking at e.g. https://www.google.com/search?q=screensaver it probably makes sense to leave "screensaver" as a single word?

File: configuration/screensaver.md (hunk 1 above)
Comment (andrum99) [reply configuration/screensaver.md:5]: The "by default" was there already, but I agree that it is not required and just adds extra verbage. I've removed it.

File: configuration/screensaver.md (hunk 2 above)
Comment (andrum99) [reply configuration/screensaver.md:20]: > Perhaps this sentence would be less ambiguous if "screen blanking" was hyphenated like "screen-blanking" ?

A hyphen would be gramatically incorrect.

File: configuration/screensaver.md (hunk 4 above)
Comment (andrum99) [reply configuration/screensaver.md:26]: I was going by lexico.com, which is the free version of the Oxford English Dictionary online. It only has an entry for screen saver as two separate words. The page also predominently used the "two words" version before my edits.

File: configuration/screensaver.md (hunk 3 above)
Comment (andrum99) [reply configuration/screensaver.md]: Fixed.

File: configuration/screensaver.md (hunk 1 above)
Comment (lurch) [reply configuration/screensaver.md:5]: Still says "By default" in the "console" section of the page.

File: configuration/screensaver.md (hunk 4 above)
Comment (andrum99) [reply configuration/screensaver.md]: Done.

File: configuration/screensaver.md (hunk 1 above)
Comment (andrum99) [reply configuration/screensaver.md]: Thanks - fixed.

**Comment by andrum99** [comment]
OK, this should be ready to roll know. I've made a couple of further changes to remove extraneous verbage and fix up some minor grammar issues.

**Comment by JamesH65** [comment]
LGTM.

**Pull request merged**